packages/unmock-core/src/service/interfaces.ts

```diff
@@ -25,6 +25,18 @@ export interface IUnmockServiceState {
   [key: string]: UnmockServiceState;
 }
 
+export interface IService {
+  schema: OASSchema;
+  hasDefinedPaths: boolean;
+  findEndpoint(endpoint: string): string | undefined;
```

Comment:
What does this do?

Reply:
Will add documentation (_copy, paste_ your doc style in #52 )